airbyte-ci/connectors/connector_ops/tests/test_utils.py

```diff
@@ -44,20 +44,35 @@ def test_init(self, connector, exists, mocker, tmp_path):
             assert isinstance(connector.metadata, dict)
             assert isinstance(connector.support_level, str)
             assert isinstance(connector.acceptance_test_config, dict)
-            assert connector.icon_path == Path(f"./airbyte-config-oss/init-oss/src/main/resources/icons/{connector.metadata['icon']}")
+            assert connector.icon_path == Path(f"./airbyte-integrations/connectors/{connector.technical_name}/icon.svg")
             assert len(connector.version.split(".")) == 3
         else:
             assert connector.metadata is None
             assert connector.support_level is None
             assert connector.acceptance_test_config is None
-            assert connector.icon_path == Path(f"./airbyte-config-oss/init-oss/src/main/resources/icons/{connector.name}.svg")
+            assert connector.icon_path == Path(f"./airbyte-integrations/connectors/{connector.technical_name}/icon.svg")
```

Comment:
I guess these were tests that weren't being run, and didn't pass after we made them run?

Reply:
exactly! I want to get them run automatically again https://github.com/airbytehq/airbyte/pull/30330#discussion_r1322258706

Ill address that in another PR

Reply:
great, thanks for fixing them in any case :D